import/controls/permissionmanager.h

```diff
@@ -0,0 +1,53 @@
+/****************************************************************************
+**
+** Copyright (c) 2020 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef PERMISSIONMANAGER_H
+#define PERMISSIONMANAGER_H
+
+#include <QObject>
+#include <QJSValue>
+#include <QScopedPointer>
+
+class Permission;
+
+class PermissionManager : public QObject
+{
+    Q_OBJECT
+public:
+    PermissionManager(QObject *parent = nullptr);
+
+    enum Capability {
+        Unknown,
```

Comment:
I would recommend to set enum values to the same as in https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsIPermissionManager and leave a comment to keep this manual synchronization in mind.

Reply:
Very good point. Add both mozilla doc ref and https://git.sailfishos.org/mer-core/gecko-dev/blob/master/netwerk/base/nsIPermissionManager.idl ref